lib/app.rb

```diff
@@ -50,9 +49,10 @@ class App < Sinatra::Base
     use Routes::Comments
     use Routes::Nits
     use Routes::Teams
+    use Routes::Sessions
```

Comment:
Do you want to put Sessions last? In your other branch you have a comment saying that "this needs to be last" (meaning the Sessions). Why does it need to be last?